src/WordsFunctionArgException.java

```diff
@@ -1,26 +1,20 @@
 
-public class WordsFunctionArgException extends Exception {
+@SuppressWarnings("serial")
+public class WordsFunctionArgException extends WordsException {
```

Comment:
I don't quite understand how this one will work.  A function (custom action) can receive multiple parameters.  How do you map a single "expectedArg" to a single "receivedArg"?  Wouldn't it instead be a WordsFunctionMissingArgException, and the exception is simply the expectedArg that was missing?